docs/src/main/asciidoc/lra.adoc

```diff
@@ -58,8 +58,17 @@ This will add the following to your pom.xml:
         <groupId>io.quarkus</groupId>
         <artifactId>quarkus-narayana-lra</artifactId>
     </dependency>
+    <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-resteasy-jackson</artifactId>
+    </dependency>
 ----
 
+[IMPORTANT]
+====
+`quarkus-narayana-lra` can work with either `quarkus-resteasy-jackson` or `quarkus-resteasy-reactive-jackson`. Without these extensions, `quarkus-narayana-lra` has no effect
```

Comment:
Wouldn't it be better if it failed with an error saying that one of these extensions is needed?

Reply:
That's probably a good idea, let me do that.

Reply:
Agreed.
And I think the easiest thing to do for now is not support reactive.

Reply:
Well, it comes for free, so no reason not to support it :)